jenkins/version-def.sh

```diff
@@ -26,10 +26,10 @@ for VAR in $OVERWRITE_PARAMS; do
 done
 IFS=$PRE_IFS
 
-CUDF_VER=${CUDF_VER:-"23.12.0"}
+CUDF_VER=${CUDF_VER:-"23.12.1"}
```

Comment:
cudf is 23.12.0?

Reply:
Yes, cuDF version is v23.12.1,  as ai/rapida/cudf native version is v23.12.01: https://repo.maven.apache.org/maven2/ai/rapids/cudf/23.12.1/

In fact we did not release cuDF JNI v23.12.0